levels/bitfs/areas/script_vanilla.inc.c

```diff
@@ -6,5 +6,3 @@ LOAD_YAY0(        /*seg*/ 0x05, _group2_yay0SegmentRomStart, _group2_yay0Segment
 LOAD_RAW(         /*seg*/ 0x0C, _group2_geoSegmentRomStart, _group2_geoSegmentRomEnd),
 LOAD_YAY0(        /*seg*/ 0x06, _group17_yay0SegmentRomStart, _group17_yay0SegmentRomEnd),
 LOAD_RAW(         /*seg*/ 0x0D, _group17_geoSegmentRomStart, _group17_geoSegmentRomEnd),
-LOAD_YAY0(        /*seg*/ 0x08, _common0_yay0SegmentRomStart, _common0_yay0SegmentRomEnd),
-LOAD_RAW(         /*seg*/ 0x0F, _common0_geoSegmentRomStart, _common0_geoSegmentRomEnd),
```

Comment:
Not technically in scope of this PR, but probably worth fixing here anyway; it seems like BitFS, WF, and THI reversed the roles of `script_vanilla` and `script_vanilla_load`! The inconsistency here should definitely be resolved.

Also apparently BoB's level segment wasn't moved to `script_vanilla_load` for some reason.